app/src/main/res/layout/listview_tent_header.xml

```diff
@@ -24,6 +24,6 @@
         android:layout_height="wrap_content"
         android:text="@string/unknown_tent"
         android:textAppearance="@style/text.heading"
-        tools:text="@string/probable" />
+        tools:text="Probable 1 (10 patients)" />
```

Comment:
ping, i`m fixing all the TODO i18n and also found even more strings to translate.